examples/dll/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use bindings::Windows::Win32::System::SystemServices::{
+    DLL_PROCESS_ATTACH, DLL_PROCESS_DETACH, DLL_THREAD_ATTACH, DLL_THREAD_DETACH, HINSTANCE,
+};
+use bindings::Windows::Win32::UI::WindowsAndMessaging::{MessageBoxA, MB_OK};
+use std::ffi::c_void;
+
+#[no_mangle]
+#[allow(non_snake_case)]
+#[allow(unused_variables)]
+pub extern "stdcall" fn DllMain(module: HINSTANCE, reason: u32, reserved: *mut c_void) -> bool {
```

Comment:
[DllMain](https://docs.microsoft.com/en-us/windows/win32/dlls/dllmain) returns a `BOOL`, which [maps to](https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types) an `int` in C, corresponding to an `i32` in Rust.